NuGet.Config

```diff
@@ -10,6 +10,7 @@
       Used for azure-sdk-tools repo until issue https://github.com/Azure/azure-sdk-tools/issues/1329 is addressed
     -->
     <add key="azure-sdk-tools" value="https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.json" />
+    <add key="myget" value="https://www.myget.org/F/azure-appservice/api/v3/index.json" />
```

Comment:
Doesn't look intentional, to confirm this will be removed?

Reply:
Yes, this is will be removed -- we are currently referencing a prerelease version of the microsoft.azure.webjobs nuget package from this internal feed.

Reply:
Removed as 3.0.36 has been released